README.rst

```diff
@@ -51,23 +73,67 @@ Transformer Engine in action:
   model = te.Linear(in_features, out_features, bias=True)
   inp = torch.randn(hidden_size, in_features, device="cuda")
 
-  # Create FP8 recipe. Note: All input args are optional.
+  # Create an FP8 recipe. Note: All input args are optional.
   fp8_recipe = recipe.DelayedScaling(margin=0, interval=1, fp8_format=recipe.Format.E4M3)
 
-  # Enables autocasting for the forward pass
+  # Enable autocasting for the forward pass
   with te.fp8_autocast(enabled=True, fp8_recipe=fp8_recipe):
       out = model(inp)
 
   loss = out.sum()
   loss.backward()
 
+
+JAX Examples
+^^^^^^^^^^^^
+
+.. code-block:: python
+
+  import jax
+  import jax.numpy as jnp
+  import transformer_engine.jax as te
+  from transformer_engine.common import recipe
+
+  BATCH = 32
+  SEQLEN = 128
+  HIDDEN = 1024
+
+  # Initialize RNG and inputs.
+  rng = jax.random.PRNGKey(0)
+  init_rng, data_rng = jax.random.split(rng)
+  inp = jax.random.normal(data_rng, [BATCH, SEQLEN, HIDDEN], jnp.float32)
+
+  # Create an FP8 recipe. Note: All input args are optional.
+  fp8_recipe = recipe.DelayedScaling(margin=0, interval=1, fp8_format=recipe.Format.HYBRID)
+
+  # Enable autocasting for the forward pass
+  with te.fp8_autocast(enabled=True, fp8_recipe=fp8_recipe):
+      model = te.DenseGeneral(features=HIDDEN)
+
+      def loss_fn(params, other_vars, inp):
+        out = model.apply({'params':params, **other_vars}, inp)
+        return jnp.mean(out)
+
+      # Initialize models.
+      variables = model.init(init_rng, inp)
+      other_variables, params = variables.pop('params')
+
+      # Construct the forward and backward function
+      fwd_bwd_fn = jax.value_and_grad(loss_fn, argnums=(0, 1))
+
+      for _ in range(10):
+        loss, (param_grads, other_grads) = fwd_bwd_fn(params, other_variables, inp)
+        # Update FP8 metas
+        other_variables = te.update_fp8_metas(other_grads)
+
+
 Highlights
 ----------
 
-* Easy-to-use pyTorch modules enabling building of the Transformer layers with FP8 support on H100
-  GPUs.
+* Easy-to-use Transformer Engine's modules enabling building of the Transformer layers with FP8 support
```

Comment:
Let's use either "pyTorch and JAX" or just "modules".

Reply:
Change to "modules".